force-app/main/utilities/schema/classes/SObjectRecordTypes.cls

```diff
@@ -1,20 +1,25 @@
 /**
- * @author Gavin Palmer (gavinhughpalmer@gmail.com)
```

Comment:
@KhinHou can you revert the email update here?

Reply:
reverted back!